### PR TITLE
Zer-80

### PR DESCRIPTION
**Pull request by olenochka123**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by halyna-m — APPROVED** [review]

**Review by andriibureviy — APPROVED** [review]

File: app/controllers/api/v1/calculators_controller.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@
 module Api
   module V1
     class CalculatorsController < ApplicationController
+      VALUES = { bought_diapers: 8956, money_spent: 7841, garbage_created: 342 }
```
Comment (maksymst) [inline app/controllers/api/v1/calculators_controller.rb:6]: add empty line after this

File: spec/requests/values_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Values', type: :request do
```
Comment (maksymst) [inline spec/requests/values_spec.rb:3]: RSpec.describe '/api/v1/calculators', type: :request do

File: spec/requests/values_spec.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    before do
+      post "/api/v1/calculators/PERMALINK/compute"
+    end
+
+    it 'returns http success' do
+      expect(response.status).to eq(200)
+    end
+
+    it 'returns JSON data correctly' do
+      expect(json_response).to eq({"bought_diapers"=>8956, "money_spent"=>7841, "garbage_created"=>342})
```
Comment (maksymst) [inline spec/requests/values_spec.rb:15]: https://github.com/rubocop/ruby-style-guide#consistent-string-literals-single-quote

File: spec/requests/values_spec.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    end
+
+    it 'JSON contains response' do
+      expect(json_response).to be_truthy
+    end
```
Comment (maksymst) [inline spec/requests/values_spec.rb:20]: what is the purpose of this test?

File: config/routes.rb [hunk 5]
```diff
@@ -11,6 +11,11 @@
     resources :calculators, only: %i[new create edit update]
   end
   namespace :api do
+    namespace :v1 do
+      resources :calculators, only: [] do
+        post :compute
```
Comment (maksymst) [inline config/routes.rb:16]: see v2 endpoint route

**Pull request merged**